src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -269,18 +272,17 @@ void testUpdateWorkspace() {
   @Test
   void testHandlesSamError() {
     String errorMsg = "fake SAM error message";
-
-    doThrow(new SamApiException(errorMsg))
-        .when(mockSamService)
-        .createWorkspaceWithDefaults(any(), any());
-
-    SamApiException exception =
+    String apiErrorMsg = "test";
+    ErrorReportException testex =
+        SamExceptionFactory.create(errorMsg, new ApiException(apiErrorMsg));
```

Comment:
nit: Should we explicitly `SamInternalServerErrorException exception = new SamInternalServerErrorException(apiErrorMessage)` instead of using the factory and relying on what class it returns?

Reply:
Or assert that the factory returns what you expect?

Reply:
Going with the former as the factory has it's own tests.